code/modules/antagonists/changeling/powers/digitalcamo.dm

```diff
@@ -4,15 +4,20 @@
 	helptext = "We cannot be tracked by camera while using this skill."
 	button_icon_state = "digital_camo"
 	dna_cost = 1
+	power_type = CHANGELING_PURCHASABLE_POWER
+
+/datum/action/changeling/digitalcamo/Remove(mob/M)
+	REMOVE_TRAIT(M, TRAIT_AI_UNTRACKABLE, CHANGELING_TRAIT)
+	..()
 
 //Prevents AIs tracking you.
 /datum/action/changeling/digitalcamo/sting_action(mob/user)
-
-	if(user.digitalcamo)
+	if(HAS_TRAIT(user, TRAIT_AI_UNTRACKABLE))
+		REMOVE_TRAIT(user, TRAIT_AI_UNTRACKABLE, CHANGELING_TRAIT)
```

Comment:
use `HAS_TRAIT_FROM` so that you can check the source is a changeling trait. incase we use this trait for something else in the future